src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -100,40 +103,46 @@ def validate_request_update_args(request_args, config, reqmgr_db_service, param)
     else:
         args_without_status = request_args
 
-    if len(args_without_status) > 0 and not workqueue_stat_validation(args_without_status):
+    if len(args_without_status) == 1 and 'RequestPriority' in args_without_status:
+        if not isinstance(args_without_status['RequestPriority'], int):
```

Comment:
Could we just try to convert and if gives value error then through the exception? Is it passing string?